packages/eslint-plugin-jest/README.md

```diff
@@ -31,6 +31,7 @@ Then configure the rules you want to use under the rules section.
     "jest/no-disabled-tests": "warn",
     "jest/no-focused-tests": "error",
     "jest/no-identical-title": "error",
+    "jest/valid-expect": "error",
```

Comment:
This line is a breaking change btw. If the next release is not a major, this should be dropped